.github/workflows/release.yml

```diff
@@ -0,0 +1,101 @@
+name: Release
+
+
+on:
+  release:
+    types: [created]
```

Comment:
In other projects we do this on "push/tags". Not sure if they differ. Might be good to either change them all to the one or the other.

Reply:
The only thing I prefer is that we do the same thing for all projects. So pick one and apply it everywhere. This is not about personal preferences, this is about keeping things sane :)